provision/acc_provision/acc_provision.py

```diff
@@ -1478,14 +1488,27 @@ def generate_kube_yaml(args, config, operator_output, operator_tar, operator_cr_
         "configmap", "secret", "serviceaccount",
         "daemonset", "deployment",
     ]
+
+    if config.get("target_version"):
+        if (parse_version.parse(config["target_version"]) <= parse_version.parse('5.2.3.3')):
+            ver = "5-2-3-3"
+        elif (parse_version.parse(config["target_version"]) < parse_version.parse('5.2.3.5')):
+            ver = "5-2-3-4"
+        elif parse_version.parse(config["target_version"]) >= parse_version.parse('5.2.3.5'):
+            ver = "5-2-3-5"
```

Comment:
we have not released 5.2.3.6 but we will be soon, should we proactively add it here?

Reply:
noted